tasks/skylab/Metrics.wdl

```diff
@@ -9,7 +9,7 @@ task CalculateCellMetrics {
 
     # runtime values
 
-    String docker = "us.gcr.io/broad-gotc-prod/warp-tools:1.0.1-1679941323"
+    String docker = "us.gcr.io/broad-gotc-prod/warp-tools:1.0.6-1692962087"
```

Comment:
i noticed this docker tag is different than the other warp-tools tags (1.0.5-1692706846). I just want to make sure this was on purpose?

Reply:
I missed that you were asking about 1.05 in the h5ad task. Thanks for catching! I updated and am retesting now.